docs/relations-fields.md

```diff
@@ -0,0 +1,145 @@
+# Relations Fields
+
+Fields in the "relations" group are used to describe how records relate to each other. All of them are designed to store the [ID(s)](https://opengeometadata.org/ogm-aardvark/#id) of related records. More than one record can be listed within each relationship field.
+
+Starting in GeoBlacklight version 4.0, most of these fields are used to populate relationship widgets in the application sidebar, allowing users to easily explore related records and understand how they are connected. The relationship widgets are customizable in the `geoblacklight.en.yml` and `settings.yml` files.
+
+
+| Field | Field Name | Obligation |
+|:------|:-----------|:-----------|
+| [Relation](../ogm-aardvark/#relation) | `dct_relation_sm` | Optional |
+| [Member Of](../ogm-aardvark/#member-of)| `pcdm_memberOf_sm` | Optional |
+| [Is Part Of](../ogm-aardvark/#is-part-of) | `dct_isPartOf_sm` | Optional |
+| [Source](../ogm-aardvark/#source) | `dct_source_sm` | Optional |
+| [Is Version Of](../ogm-aardvark/#is-version-of) | `dct_isVersionOf_sm` | Optional |
+| [Replaces](../ogm-aardvark/#replaces) | `dct_replaces_sm` | Optional |
+| [Is Replaced By](../ogm-aardvark/#is-replaced-by) | `dct_isReplacedBy_sm` | Optional |
+
+## Relation
+
+!!! warn
```

Comment:
Change to "warning" here instead of "warn" to trigger the orange admonition with the explanation mark (https://squidfunk.github.io/mkdocs-material/reference/admonitions/#type:warning)

Reply:
Ooh, great, thanks!